.github/workflows/ci-sharkfuser.yml

```diff
@@ -108,8 +112,12 @@ jobs:
       working-directory: ${{ env.SHARKFUSER_DIR }}
       run: |
         ctest --test-dir build -T test -T coverage --timeout 30
-        lcov --capture --directory build --output-file build/coverage.info
-        lcov --remove build/coverage.info 'build/*' '/usr/*' --output-file build/coverage.info
+        # LLVM includes some broken .gcda files to test llvm-cov. There doesn't
+        # seem to be a way to convince lcov to not scan the broken files, only
+        # to ignore them after the scan, hence the `--ignore-errors` flag as
+        # well as the `--exclude`.
+        lcov --capture --directory build --ignore-errors gcov,version --exclude "*/_deps/*" --output-file build/coverage.info
+        lcov --remove build/coverage.info 'build/*' '/usr/*' --ignore-errors unused --output-file build/coverage.info
```

Comment:
@sjain-stanford the `.gcda` files in llvm reak gcov-viewer as well.

Reply:
We could do something like delete them all from the LLVM checkout in CMake... but that sounds a little error prone.

Reply:
We have some directory exclusions for code coverage in `build_tools/cmake/CTestCustom.cmake.in` - this is specifically to avoid generating `.gcda` / `.gcno` files. Does adding to it help? Or are you saying these files are already available in LLVM so they always get picked by lcov?

Reply:
Also, is gcov-viewer not usable after bringing in LLVM? We should find a way to make it work.

Reply:
> We have some directory exclusions for code coverage in build_tools/cmake/CTestCustom.cmake.in - this is specifically to avoid generating .gcda / .gcno files. Does adding to it help? Or are you saying these files are already available in LLVM so they always get picked by lcov?

They are test fixtures in LLVM for testing `llvm-cov` that are there in the repo on checkout, not generated by the coverage run, so I don't think the exclusions in CTestCustom.cmake.in do anything in this case.

Reply:
> Also, is gcov-viewer not usable after bringing in LLVM? We should find a way to make it work.

Yes it's broken after bringing in LLVM. I'm not sure how to fix it besides deleting the text fixtures from the LLVM checkout. I tried a `.lcovrc` file and it seems to just be ignored.